config-example/cl/config.yaml

```diff
@@ -57,7 +73,7 @@ EIP7805_FORK_EPOCH: $EIP7805_FORK_EPOCH
 # 12 seconds
 SECONDS_PER_SLOT: $SLOT_DURATION_IN_SECONDS
 # 14 (estimate from Eth1 mainnet)
-SECONDS_PER_ETH1_BLOCK: $SLOT_DURATION_IN_SECONDS
+SECONDS_PER_ETH1_BLOCK: 14
```

Comment:
this is a mistake? i mean we dont need the value anymore technically after pectra - but it is basically the slot time of the chain. It was 14s in PoW world and we never modified it since it made no big difference

Reply:
I'm using the default value from mainnet.yaml from consensus specs. Wanna keep this value the same.

Reply:
fixed.